ecl/eclcc/eclcc.cpp

```diff
@@ -1976,7 +1976,12 @@ bool EclCC::parseCommandLineOptions(int argc, const char* argv[])
         else if (strcmp(arg, "-internal")==0)
         {
             outputSizeStmts();
-            testHqlInternals();
+            int error = testHqlInternals() + testReservedWords();  //NOTE: testReservedWords() depends on testHqlInternals() so must be be called after.
+            // report test result
+            if (error)
+                printf("%d error%s found!\n", error, error<=1?"":"s");
+            else
+                printf("No errors\n");
```

Comment:
I wouldn't have thought so as these are just print to screen. This is the same functionality when using '-internal' as before this change, I simply moved this out from testHqlInternals(). If I was to  guess, since it doesn't fail and you can still pass it an input file, that Gavin maybe occasionally uses it for the mem sizes when making changes, - long shot?
So are we perhaps wanting this to no longer be transparent? If so, it may not even be worth it being translucent (only changing eclcc return value) and instead, make it opaque and actually fail?